tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -329,9 +336,17 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		gomega.Expect(isDiskAttached).To(gomega.BeTrue(), "Volume is not attached")
 
 		ginkgo.By("Verify the volume is accessible and available to the pod by creating an empty file")
-		filepath := filepath.Join("/mnt/volume1", "/emptyFile.txt")
-		_, err = framework.LookForStringInPodExec(namespace, pod.Name, []string{"/bin/touch", filepath}, "", time.Minute)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		if windowsEnv {
+			time.Sleep(sleepTimeOut * time.Second)
```

Comment:
There should not be any static wait time. Add mechanism to wait untill the file is available